api/server/server_unit_test.go

```diff
@@ -1,104 +0,0 @@
-package server
-
-import (
-	"encoding/json"
-	"fmt"
-	"io"
-	"net/http"
-	"net/http/httptest"
-	"testing"
-
-	"github.com/docker/docker/api"
-	"github.com/docker/docker/engine"
-	"github.com/docker/docker/pkg/version"
-)
-
-func TestHttpError(t *testing.T) {
```

Comment:
Didn't yet reimplemented this because...what we are testing here? the go http package?